webapp-mgmt/cas-management-webapp/build.gradle-bak

```diff
@@ -0,0 +1,103 @@
+apply plugin: "war"
```

Comment:
What is the responsibility of this file? Should it be removed?

Reply:
It was the old build under cas-management-webapp that I am attempting to move to gradle/webapp.gradle to match CAS build

Reply:
Kept it around to pull things out fif needed.  It will eventually be deleted.